functors task (1)/student/include/impl.h

```diff
@@ -40,9 +45,16 @@ struct MixingPolicy
      * Whiskey + SevenUp = SevenPlusSeven;
      * Others + Others = Oops;
      */
-    static Cocktail mix(/**???*/)
+    static Cocktail mix(AlcoholDrink alcoholDrink, NonAlcoholDrink nonAlcoholDrink)
     {
+        if (alcoholDrink == AlcoholDrink::Gin && nonAlcoholDrink == NonAlcoholDrink::LimeJuice)
```

Comment:
Please fix formatting!
Always use braces after loop/if:
https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces

Reply:
In general it is true.

But what about different coding styles which allow you (and even encourage) to do so?
https://google.github.io/styleguide/cppguide.html#Conditionals
https://webkit.org/code-style-guidelines/#braces-one-line
https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements
Google, WebKit, LLVM... they are not so stupid to miss that discussion on stack overflow. What do you think?

Yes, I'll fix it. But in general, if we don't have dedicated coding style for this training, I'd say it is not the best idea to say "always" here.